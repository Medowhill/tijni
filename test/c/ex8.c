void f(int *p, int x) {
  *p = x;
}

void g(int *p, int x) {
  f(p, x);
}
