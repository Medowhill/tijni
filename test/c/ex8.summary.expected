{
  "f": {
    "params": [ { "name": "%p(f)", "used": true, "freed": false }, { "name": "%x(f)", "used": false, "freed": false } ],
    "ret": [],
    "env": {
      "%p(f)": [ "'%p(f)" ],
      "%x(f)": [ "'%x(f)" ],
      "'%p(f)": [ "'%x(f)" ]
    }
  },
  "g": {
    "params": [ { "name": "%p(g)", "used": true, "freed": false }, { "name": "%x(g)", "used": false, "freed": false } ],
    "ret": [],
    "env": {
      "%p(f)": [ "'%p(g)" ],
      "%x(f)": [ "'%x(g)" ],
      "%p(g)": [ "'%p(g)" ],
      "%x(g)": [ "'%x(g)" ],
      "'%p(g)": [ "'%x(g)" ]
    }
  }
}